earthmover/nodes/node.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, name: str, config: 'YamlMapping', *, earthmover: 'Earthmover'
         self.error_handler: 'ErrorHandler' = earthmover.error_handler
 
         self.error_handler.ctx.update(
-            file=self.config.__file__, line=self.config.__line__, node=self, operation=None
+            file=self.config.get("__file__",""), line=self.config.get("__line__",0), node=self, operation=None
```

Comment:
To circumvent needing to change these lines, we should just initialize the No-Op config block as a `YamlMapping` and set default values for the `__file__` and `__line__` attributes in the class.

Reply:
But what would those default values be? (when nodes are added programmatically, not from a YAML configuration file)

Can `__file__` and `__line__` be `None`? That's the only default value I think would make sense, but I don't know how it would come through in the logging messages, or if it would cause errors.